src/Catalog.js

```diff
@@ -1,42 +1,41 @@
-import React from 'react';
+import React from "react";
 
-const Catalog = (props) => {
+const Catalog = props => {
   return (
     <ul className="phones">
-
-      { props.phones.map(phone => (
-        <li className="thumbnail" key={phone.id} >
+      {props.phones.map(phone => (
+        <li className="thumbnail" key={phone.id}>
           <a
-            href={'#' + phone.id}
+            href={"#" + phone.id}
```

Comment:
```suggestion
            href={`#${phone.id}`}
```
Template string is powerful feature